README.md

```diff
@@ -1,24 +1,89 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+## Description
 
-* Ruby version
+In this project, I created a budget website in which users can track their finances and what they're being spent on. 
 
-* System dependencies
+## Screenshots
 
-* Configuration
+![image](https://user-images.githubusercontent.com/74814780/159991267-1dcae1b2-f4a4-4837-be7a-dc4877926c5b.png)
 
-* Database creation
+![image](https://user-images.githubusercontent.com/74814780/159991298-b0889634-e711-4676-89e0-fc7f0d271367.png)
 
-* Database initialization
+## Built with
 
-* How to run the test suite
+- Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+- HTML
 
-* Deployment instructions
+- CSS
 
-* ...
+- Linters(Rubocop and Stylelint)
+
+## Live Link
+[Demo Link](http://127.0.0.1:3000/users/sign_up)
+
+[Presentation Video](https://www.loom.com/share/b04c99be722645e9a9621f127753e7b4)
```

Comment:
- [ ] Kindly ensure that the video presentation contains **highlights about pieces of the code that you find interesting** or is proud of making, as requested in the previous code review.

Reply:
Hi @rloterh , I re-recorded a new video after getting feedback from the first reviewer and highlighted pieces of code that I was proud of implementing towards the end of the video. I mentioned that "I'm proud of the fact I was able to let users add amount and transactions"

Reply:
@Ugboaja-Uchechi 
Oh, we actually wanted you to talk briefly about some aspects of your code of this project, either via your code editor or your code on Github repository

Cheers! 😃

Reply:
OK, cool. I would do that, thank you for the clarification.